### PR TITLE
Proposal first draft

### PR DESCRIPTION
**Pull request by bls1999**
My project proposal is contained herein. I'm looking forward to seeing your proposals, and for your feedback on mine.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+
+## Title
+Study Buddy *(working title)*
+
+## The Problem
+Across the United States, millions of children lack access to basic resources such as the internet, a printer, or even a computer. This can severely hinder their ability to receive the same quality of education as that of their classmates.
+
+## The Solution
+*Study Buddy* is a way for students to ensure they can always find the resources they need with just a quick search. The app will be a hub for finding organizations that provide services like computer/internet usage, printing, copying, etc. Organizations will also be able to "claim" their profiles to provide a more accurate description of their services.
+
```
Comment (abigailzhou03) [inline README.md:12]: maybe add a phrase about how the app can motivate organizations to be more student friendly bc the organizations want to promote themselves and students will review/rate them

**Comment by BrandonSomoza** [comment]
Love how detailed yet concise the proposal is. Great format as well.
The only thing I would change is introducing the problem through a feasible scenario rather than just stating the problem.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,18 @@
-[![Open in Visual Studio Code](https://classroom.github.com/assets/open-in-vscode-c66648af7eb3fe8bc4f294546bfd86ef473780cde1dea487d3c4ff354943c9ae.svg)](https://classroom.github.com/online_ide?assignment_repo_id=8391369&assignment_repo_type=AssignmentRepo)
-# Project Proposal
-Delete the contents of this file and replace with your project proposal, as described in the [instructions](./instructions.md)
\ No newline at end of file
+# BABS Project Proposal
+*by: <u>**B**</u>en Schwartz, <u>**A**</u>bigail Zhou, <u>**B**</u>randon Samoza, <u>**S**</u>hay Gyawali*
```
Comment (BrandonSomoza) [inline README.md:2]: change Samoza to Somoza. An honest mistake, you'd be surprised how badly my name has been butchered lol

File: README.md (hunk 1 above)
Comment (abigailzhou03) [reply README.md:12]: Organizations can "claim" their profiles to promote their businesses and interact with the student's rating system, which can encourage them to become more student friendly.

File: README.md (hunk 2 above)
Comment (bls1999) [reply README.md:2]: noooooooooo

sorry, you have to change your name now. I don't make the rules :(

File: README.md (hunk 1 above)
Comment (bls1999) [reply README.md:12]: I'm thinking we can move this edit to after we mention the review system for the first time. How about we consider merging this now and then we can make final edits later?

**Review by abigailzhou03 — APPROVED** [review]

**Review by BrandonSomoza — APPROVED** [review]
Let's get to merging!

File: README.md (hunk 1 above)
Comment (abigailzhou03) [reply README.md:12]: yes!

**Pull request merged**